playground/AspireEventHub/README.md

```diff
@@ -0,0 +1,7 @@
+# EventHubs + Aspire
+
+A project showing how to use the new `AzureBicepResource` to deploy resources that don't exist in aspire by default. 
+
+This is using a [preview 4 build of .NET Aspire](https://github.com/dotnet/aspire/blob/main/docs/using-latest-daily.md).
```

Comment:
```suggestion
```

This uses the code that is in the repo. This line is wrong and is just going to get out of date. Can be removed.